modules/standard/BigInteger.chpl

```diff
@@ -2505,22 +2511,25 @@ When ``numer/denom`` does not produce an integer, this method may produce incorr
       ret = mpz_divisible_ui_p(t_.mpz,   div_);
     }
 
-    return ret.safeCast(int);
+    if ret.safeCast(int) then 
+      return true;
+    else 
+      return false;
   }
 
   /*
     .. warning::
 
-       b is deprecated - please use exp
+       bigint.divisible_p is deprecated, use bigint.isDivisible instead
   */
   pragma "last resort"
   deprecated
-  "b is deprecated - please use exp"
-  proc bigint.divisible_2exp_p(b: integral) : int {
-    return this.divisible_2exp_p(exp=b);
+  "bigint.divisible_2exp_p is deprecated, use bigint.isDivisibleBy2exp instead"
+  proc bigint.divisible_2exp_p(b: integral) : bool {
+    return this.isDivisibleBy2exp(exp=b);
   }
 
-  proc bigint.divisible_2exp_p(exp: integral) : int {
+  proc bigint.isDivisibleBy2exp(exp: integral) : bool {
```

Comment:
Hmm, I'm not sure I like this name.  I think these new names would probably be worth running past the mailing list and might suggest `isDivisibleByPowerOf2` instead?  It's a bit of a mouthful, though . . . maybe others will have better suggestions?

Personally I like `isDivisibleBy` better than `isDivisible`, but I also tend to be a bit verbose (and would be interested how those on the mailing list felt)

Reply:
I have created a new topic for the same.
[Renaming of divisible_2exp_p and congruent_2exp_p](https://chapel.discourse.group/t/renaming-of-divisible-2exp-p-and-congruent-2exp-p/8594?u=tiger-yash)

Reply:
I have given the name `isDivisibleByPowerOf2` for the time being..

Reply:
I guess `isDivisibleBy2Pow` is the favoured name. 
Should I change it to that @lydia-duncan ?